internal/pkg/override/override.go

```diff
@@ -59,31 +104,31 @@ func (ext extension) isYAML() bool { return ext == ".yml" || ext == ".yaml" }
 
 func lookupYAMLPatch(path string, fs afero.Fs) (Info, error) {
 	if ext := extension(filepath.Ext(path)); !ext.isYAML() {
-		return unknownOverrider, fmt.Errorf("YAML patch documents require a .yml or .yaml extension: %q has a %q extension", path, ext)
+		return Info{}, fmt.Errorf("YAML patch documents require a .yml or .yaml extension: %q has a %q extension", path, ext)
```

Comment:
How about keeping extension values in double quotes? 
```suggestion
		return Info{}, fmt.Errorf(`YAML patch documents require a ".yml" or ".yaml" extension: %q has a %q extension`, path, ext)
```

Reply:
Updated!